### DISCUSSION TITLE
[Feature] Media Channels for Collection Separation

### DISCUSSION THREAD

**Discussion by kuzmandi** [immich-app/immich#13866] [Feature Request]
Dear folks!

I have a photo collection that's divided into "channels," meaning it's essentially multiple collections that should be viewed separately. Above all, the images shouldn’t mix when they’re listed and viewed by date, geolocation, content, etc.

I see several approaches to achieving this with Immich:

1.) There would need to be an option to completely hide images with specific tags. This would allow me to view multiple "channels" together or separately. For example, I could view only family photos, private trips, commissioned images, some together or all at once.

2.) I'm not sure if anyone else has a similar setup where there are multiple sections within the entire collection that they want to separate (not just "private" and "public"). If so, assigning such sections (which I’ve called channels) would be the most ideal solution.

3.) I could also manage this workaround by using multiple users. But for this to be practical for me, I would need a quick switch between users without a full log-out and log-in process. If I could just click on the user at the top right and select another one (so several users would be logged in, and I could toggle between them), that would also be a solution.

Is such a feature possible in the near future?

Please let me know your thoughts. Maybe I've overlooked a possible solution...

Best regards,
Andreas

**Comment by bo0tzz** [comment]
#3150
